.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+/*/target
```

Comment:
Could probably just be `**/target`

Reply:
I don't really like doing `**` in `.gitignore`, it's too broad.